cmd/sonobuoy/app/status.go

```diff
@@ -75,9 +74,23 @@ func getStatus(cmd *cobra.Command, args []string) {
 	for _, pluginStatus := range status.Plugins {
 		fmt.Fprintf(tw, "%s\t%s\t%s\n", pluginStatus.Plugin, pluginStatus.Node, pluginStatus.Status)
 	}
-	fmt.Fprintf(tw, "\t\t%s\n", strings.ToUpper(status.Status))
+
 	if err := tw.Flush(); err != nil {
 		errlog.LogError(errors.Wrap(err, "couldn't write status out"))
 		os.Exit(1)
 	}
+	fmt.Printf("\n%s\n", statusAsString(status.Status))
+}
+
+func statusAsString(str string) string {
+	switch str {
+	case "running":
+		return "Sonobuoy is still running. Runs can take up to 60 minutes."
```

Comment:
Only if it's conformance or extended does it take that long.

Reply:
@timothysc we haven't rolled anything up yet